docs_src/design/adr/security/0017-consul-security.md

```diff
@@ -96,25 +96,25 @@ Phase 3 will not introduce any breaking changes on top of the Phase 2 breaking c
   with the global-management policy and install it into the agent.
   During phase 2, it will be changed to a specific, limited, policy.
   (This change should not be visible to Consul API clients.)
-- Create a global management token (stored in a file) for use in remote configuration.
-  This token will be used by the API gateway for proxy authentication,
+- The bootstrap management token will also be stored persistently
+  to be used by the API gateway for proxy authentication,
```

Comment:
The policy for a separate magmt token will need to be figured out in phase 3 and thus we are using the same Consul's bootstrap token for both security bootstrapper' s bootstrapping and kong's proxy setup

Reply:
Updated.